package.json

```diff
@@ -9,16 +9,26 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@mdx-js/loader": "^2.1.5",
+    "@mdx-js/react": "^2.1.5",
+    "@next/mdx": "^13.0.4",
```

Comment:
Also these dependencies you are not using directly, even if I understand `@next/` to be the official prefix of optional `next` plugins.
However, maybe we should audit the dependencies at some point...

Reply:
I think this is now solved.